app/controllers/glimr_cases_controller.rb

```diff
@@ -0,0 +1,17 @@
+class GlimrCasesController < ApplicationController
+
```

Comment:
Can you take the whitespace out here? I don't know why Rubocop is not picking up on this 😕